src/app/dashboard/[subdomain]/editor/page.tsx

```diff
@@ -348,9 +350,22 @@ export default function SubdomainEditor() {
       try {
         if (
           !file.type.startsWith("image/") &&
-          !file.type.startsWith("audio/")
+          !file.type.startsWith("audio/") &&
+          !file.type.startsWith("video/")
         ) {
-          throw new Error("You can only upload images and audios")
+          throw new Error("You can only upload images, audios and videos")
+        }
+
+        const uploadFilesize = filesize(file.size).to("MB")
```

Comment:
What is the function of `filesize` here, why not just use `file.size`?

Reply:
https://www.npmjs.com/package/file-size
This package provides storage unit conversion. Convert `bytes` to `mb` here.

Reply:
The`filesize` package is 1.6 kb. Whether bytes needs to be manually evaluated like `file.size / 1024 / 1024`?